specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/streamingservice.json

```diff
@@ -1093,50 +1140,50 @@
       "properties": {
         "description": {
           "type": "string",
-          "description": "The description of the Live Output."
+          "description": "The description of the live output."
         },
         "assetName": {
           "type": "string",
-          "description": "The asset name."
+          "description": "The asset that the live output will write to."
         },
         "archiveWindowLength": {
           "format": "duration",
           "type": "string",
-          "description": "ISO 8601 timespan duration of the archive window length. This is duration that customer want to retain the recorded content."
+          "description": "ISO 8601 timespan duration of the archive window length. This is the maximum content length that can be archived in the asset for this live output, also the maximum content length for the rewind window."
```

Comment:
Maybe include an ISO 8601 sample here?  Not a lot of folks know that format or are too lazy to look it up and find the part that include PTHHMMSS

Reply:
"Example: PT1H to indicate one hour."